From df5cf549fd4de266b873f41333e6e049b03c9bc5 Mon Sep 17 00:00:00 2001
From: Sami Liedes <sami.liedes@iki.fi>
Date: Mon, 20 Jan 2014 05:49:52 +0200
Subject: [PATCH 2/6] Fix a build failure on Debian unstable.

<fdt.h> needs fdt{16,32,64}_t defined in libfdt_env.h, so add typedefs
for these.

Patch adapted from
http://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/?id=ea6e7975964c75e0c8224205e0c1fd04ba37ed4a

Signed-off-by: Sami Liedes <sami.liedes@iki.fi>
---
 qemu/libfdt_env.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/qemu/libfdt_env.h b/qemu/libfdt_env.h
index 90d7f3b..45aca8c 100644
--- a/qemu/libfdt_env.h
+++ b/qemu/libfdt_env.h
@@ -21,6 +21,10 @@
 
 #include "bswap.h"
 
+typedef uint16_t fdt16_t;
+typedef uint32_t fdt32_t;
+typedef uint64_t fdt64_t;
+
 #ifdef HOST_WORDS_BIGENDIAN
 #define fdt32_to_cpu(x)		(x)
 #define cpu_to_fdt32(x)		(x)
-- 
1.9.0

